src/Profile/InfoBoxConfig.cpp

```diff
@@ -127,6 +127,10 @@ Profile::Load(const ProfileMap &map, InfoBoxSettings &settings)
     break;
   }
 
+  map.Get(ProfileKeys::InfoBoxTitleScale, settings.scale_title_font);
+  if ((settings.scale_title_font < 80) || (settings.scale_title_font > 150))
```

Comment:
Unnecessary parantheses
What's this "80" value?

Reply:
I will fix this.

Replace 80 by 50 to make it consistent throughout the changes.